views/moderation.js

```diff
@@ -109,6 +110,15 @@ module.exports = function (cabal, modKey, db) {
 
         return readonly(out)
       },
+
+      banInfo: function (core, feedSeq, cb) {
+        if (typeof feedSeq === 'string') {
```

Comment:
This would benefit from being wrapped in a `this.ready()` (like `isBanned` below), so it waits for the indexes to catch up first.